packages/valory/services/oracle_hardhat/service.yaml

```diff
@@ -16,8 +16,9 @@ setup_args:
       safe_contract_address: []
     observation_interval: 30
     broadcast_to_server: false
-    service_registry_address: '0x0DCd1Bf9A1b36cE34237eEaFef220932846BCD82'
+    service_registry_address: ${SERVICE_REGISTRY_ADDRESS:str:0x998abeb3E57409262aE5b751f60747921B33613E}
```

Comment:
why did the default change?

Reply:
We cannot use the default hardhat image anymore because it does not have the registry contracts deployed and we cannot use the staging chain anymore this address points to the service registry on staging chain. `valory/autonolas-registries` have these contracts deployed so we use it as the local hardhat deployment, and it has the service registry deployed on the updated address

Reply:
Ok